packages/manager/modules/pci/src/projects/project/storages/databases/databases.html

```diff
@@ -68,14 +68,19 @@
             data-sortable
             data-filterable
         >
-            <img
-                ng-if="$ctrl.ENGINE_LOGOS[$row.engine]"
-                class="database-engine-logo"
-                data-ng-src="{{:: $ctrl.ENGINE_LOGOS[$row.engine]}}"
-                alt="{{:: $row.engine}}"
-            />
-            <span data-ng-bind=":: $row.getEngineLabel()"></span>
-            <span data-ng-bind=":: $row.version"></span>
+            <span class="d-flex align-items-center">
+                <img
+                    ng-if="$ctrl.ENGINE_LOGOS[$row.engine]"
+                    class="database-engine-logo mr-2"
+                    data-ng-src="{{:: $ctrl.ENGINE_LOGOS[$row.engine]}}"
```

Comment:
```suggestion
                    data-ng-if="$ctrl.ENGINE_LOGOS[$row.engine]"
                    class="database-engine-logo mr-2"
                    data-ng-src="{{:: $ctrl.ENGINE_LOGOS[$row.engine]}}"
```

Or no `data-`prefix, but please keep consistency

Reply:
Added here: https://github.com/ovh/manager/pull/6383/commits/4b95e7da6d43820992c0e899e5af27fd08aa49fe